tests/unit/test_resolver.py

```diff
@@ -28,21 +28,40 @@
     ],
 )
 
+NPM_OUTPUT = RequestOutput.from_obj_list(
+    components=[Component(type="library", name="eggs", version="1.0.0")],
+    environment_variables=[
+        EnvironmentVariable(name="CHROMEDRIVER_SKIP_DOWNLOAD", value="true", kind="literal"),
+    ],
```

Comment:
Off-topic for this PR, but this just reminded me that we need to re-evaluate the env. vars that Cachito used to provide for npm. (I'm about 60% convinced that they're not a good idea)

@lkolacek are env vars part of the npm epic?

Reply:
yes we should re-evaluate the env. vars from Cachito, I assumed it will be done within other stories from epic [STONEBLD-594](https://issues.redhat.com/browse/STONEBLD-594). Personally I did not need to setup any env. vars when I tried to provide npm packages to container.

Reply:
The [current env vars](https://github.com/containerbuildsystem/cachito/blob/4bce2e75971cb4a0e768afdd1485fb04d2dc856d/cachito/workers/config.py#L27-L32) are all very niche, if you didn't try to install Chromedriver, Cypress, Geckodriver or Sass you would not have seen their effect

I don't think handling these env vars would come up naturally in any of the existing stories

Reply:
In that case I'll file a new story to revisit env vars from Cachito